catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/ops/IcebergTableOps.java

```diff
@@ -140,6 +148,32 @@ public void close() throws Exception {
     if (catalog instanceof AutoCloseable) {
       // JdbcCatalog need close.
       ((AutoCloseable) catalog).close();
+
+      // Does IcebergCatalog also use PG here?
```

Comment:
@FANNG1 can you please review this?

Reply:
yes, Iceberg JDBC backend maybe Mysql or PG.